docs/api-docs/getting-started/authentication.md

```diff
@@ -1,30 +1,90 @@
 
-BigCommerce has five different APIs that let you manage store data, log in customers, make client-side queries for product information, and more. Each requires a different authentication method.
+## Class I: Store Management REST APIs
 
-## REST APIs
+BigCommerce's Store Management REST APIs use OAuth API account access tokens to authenticate requests. Create a store management OAuth API account to generate and manage access tokens, and pass the access token in the header of the request you want to authenticate.
 
-Requests to BigCommerce's V2 and V3 REST APIs require you to pass an access token in the header. For instructions on generating this credential, see [Obtaining Store API Credentials](/api-docs/getting-started/authentication/rest-api-authentication#obtaining-store-api-credentials#obtaining-store-api-credentials).
+For more details, see [Obtaining Store API Credentials](/api-docs/getting-started/authentication/rest-api-authentication#obtaining-store-api-credentials).
 
-## Storefront API
-The Storefront API is unauthenticated, allowing you to make client-side requests for carts, checkouts, and orders using JavaScript. 
+```http title="Store management REST authentication header"
+X-Auth-Token: {{access_token}}
+```
 
+<!-- theme: info -->
+> Legacy API accounts used HTTP basic authentication. They are no longer available to new stores. [Learn more about migrating](/api-docs/getting-started/authentication/rest-api-authentication#migrating-from-legacy-to-oauth). 
 
-## GraphQL Storefront API
-There are two ways to authenticate with the GraphQL API:
-1. Via a [Storefront API token](/api-reference/cart-checkout/storefront-api-token/api-token/createtoken) passed in your request's header.
-2. Passing a token from within a Stencil theme in your request's header.
+## Class II: GraphQL API
 
-For more details, see [GraphQL API Authentication](/api-docs/storefront/graphql/graphql-storefront-api-overview#authentication).
+GraphQL API queries use JWT-style bearer tokens to authenticate. Consult the REST token generation endpoint that corresponds to your GraphQL use case to determine the required OAuth scope before you create a store management OAuth API account to request JWTs for your GraphQL queries. Pass the bearer token in the header of the query you want to authenticate. 
 
-## Customer Login API
+For more details, see [GraphQL API Authentication](/api-docs/storefront/graphql/graphql-storefront-api-overview#authentication) and [Obtaining Store API Credentials](/api-docs/getting-started/authentication/rest-api-authentication#obtaining-store-api-credentials).
 
-The Customer Login API requires authentication via a [JWT token](https://jwt.io/) and your app's OAuth client ID.
+```http title="GraphQL authentication header"
+Authorization: Bearer {{generated_jwt}}
+```
 
-For details, see [Customer Login API](/api-docs/customers/customer-login-api).
+## Class III: App APIs

```

Comment:
This section confuses me, I don't think "App APIs" are different from "Store Management APIs".

There are separately some JWT-based APIs like Current Customer API and Customer Login API, but I'm not sure they deserve a callout in a section about authentication of API requests generally.

Reply:
@bookernath I believe that the Provider APIs must be used through apps that partners create. They can't just be implemented on a store with rando store management API credentials. At least, that's what our narrative docs say. @hatertron3000 -- is this right, or do we need to update the narrative docs?

I will modify to clarify that apps can also use all the Store Mgmt APIs and rework these sections for flow. I'm also not married to the Classes -- we'll see how that shakes out.

JWT-based APIs absolutely deserve their own callout in an overview about authentication. Not only do CC and CL use JWTs, but per [my slack question about client IDs](https://bigcommerce.slack.com/archives/C071LJPGX/p1648564063603979), they also require HTTP headers with the client ID to authenticate.

Reply:
rewritten.